tests/testthat/test-update_dockerfile_packages.R

```diff
@@ -0,0 +1,10 @@
+test_that("updates a dockerfile with the given R packages file", {
```

Comment:
@jacobvjk, play with this test code to try figure out how it works. 
You can run it interactively and also via `devtools::test()` (or control + shift + T)